Diagnostics/FreeBusyChecker/Functions/OnPremDAuthFunctions.ps1

```diff
@@ -0,0 +1,629 @@
+        AddSettingToList -list $settingsList -name "TarGetOwAUrl" -value "TarGetOwAUrl Is Blank. Can also be configured to be $($standardValues[0]) or $($standardValues[1])" -color "green"
+    } elseif ($OrgRelParameter.TarGetOwAUrl -in $standardValues) {
+        AddSettingToList -list $settingsList -name "TarGetOwAUrl" -value "TarGetOwAUrl Is $($OrgRelParameter.TarGetOwAUrl). This is a possible standard value." -color "green"
+    } else {
+        $countOrgRelIssues++
+    }
+    # TarGetSharingEpr
+    if ([string]::IsNullOrWhitespace($OrgRelParameter.TarGetSharingEpr) -or $OrgRelParameter.TarGetSharingEpr -eq "https://outlook.office365.com/EWS/Exchange.asmx") {
+        AddSettingToList -list $settingsList -name "TarGetSharingEpr" -value "TarGetSharingEpr Is ideally blank. If set, should be Office 365 EWS endpoint. Example: https://outlook.office365.com/EWS/Exchange.asmx" -color "green"
+    } else {
+        AddSettingToList -list $settingsList -name "TarGetSharingEpr" -value "TarGetSharingEpr Should be blank or https://outlook.office365.com/EWS/Exchange.asmx. If set, should be Office 365 EWS endpoint." -color "red"
+        $countOrgRelIssues++
+    }
+    # FreeBusyAccessScope
+    if ([string]::IsNullOrWhitespace($OrgRelParameter.FreeBusyAccessScope)) {
+        AddSettingToList -list $settingsList -name "FreeBusyAccessScope" -value "FreeBusyAccessScope Is blank, this is the standard Value." -color "green"
+    } else {
+        AddSettingToList -list $settingsList -name "FreeBusyAccessScope" -value "FreeBusyAccessScope Should be Blank, that is the standard Value." -color "red"
+        $countOrgRelIssues++
+    }
+    # TarGetAutoDiscoverEpr
+    $OrgRelTarGetAutoDiscoverEpr = $OrgRelParameter.TarGetAutoDiscoverEpr
+    if ([string]::IsNullOrWhitespace($OrgRelTarGetAutoDiscoverEpr)) {
+        $OrgRelTarGetAutoDiscoverEpr = "Blank"
+    }
+    if ($OrgRelParameter.TarGetAutoDiscoverEpr -like "https://AutoDiscover-s.outlook.com/AutoDiscover/AutoDiscover.svc/WSSecurity") {
+        AddSettingToList -list $settingsList -name "TarGetAutoDiscoverEpr" -value "TarGetAutoDiscoverEpr Is correct" -color "green"
+    } else {
+        AddSettingToList -list $settingsList -name "TarGetAutoDiscoverEpr" -value "TarGetAutoDiscoverEpr Is not correct. Should be https://AutoDiscover-s.outlook.com/AutoDiscover/AutoDiscover.svc/WSSecurity" -color "red"
+        $countOrgRelIssues++
+    }
+    # Enabled
+    if ($OrgRelParameter.enabled -like "True") {
+        AddSettingToList -list $settingsList -name "Enabled" -value "Enabled is set to True" -color "green"
+    } else {
+        AddSettingToList -list $settingsList -name "Enabled" -value "Enabled is set to False. This may be intentional if Hybrid Free Busy lookups are done with OAuth and Intra Organization Connector." -color "yellow"
+        $countOrgRelIssues++
+    }
+    # Display the settings list
+    if ($countOrgRelIssues -eq '0') {

```

Comment:
We shouldn't be doing this; you are comparing an `int` to a `string`. PowerShell likely works, but this is a bad habit.

Reply:
Changed to "if ($countOrgRelIssues -eq  0)"